make/ingress/run

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-if [ -z "${INGRESS_CONTROLLER}" ]; then
+set -o nounset
```

Comment:
If you're going back to edit this later, maybe `-o errexit`? Not a big deal though.